test/config.spec.js

```diff
@@ -13,13 +13,13 @@ describe('Task configuration', () => {
     it(`${task}: only has pre-defined establishment permissions`, () => {
       const permissionTypes = get(permissions, task);
       permissionTypes.forEach(type => {
-        if (type.match(/^estabishment:(.*)$/)) {
+        if (type.match(/^establishment:(.*)$/)) {
           const level = type.split(':')[1];
           if (level === 'role') {
             const roleType = type.split(':')[2];
-            assert.ok(roles.includes(roleType), `${roleType} should be included in defined roles`);
+            assert.ok([ ...roles, '*' ].includes(roleType), `${roleType} should be included in defined roles`);
```

Comment:
The named-role-based permissions don't currently support `*` wildcards, so shouldn't be added to this test case.
```suggestion
            assert.ok(roles.includes(roleType), `${roleType} should be included in defined roles`);
```